mediaflow_proxy/extractors/uqload.py

```
@@ -0,0 +1,41 @@
+    async with httpx.AsyncClient(proxy=settings.proxy_url if use_request_proxy else None) as client:
+        try:
+            response = await client.get(d, follow_redirects=True)
+            response.raise_for_status()
+            
+            # Look for video URL in response using a more robust pattern
+            video_url_match = re.search(r'sources:\s*\[(["\'])(.*?)\1\]', response.text)
+            if video_url_match:
+                final_url = video_url_match.group(2)
+            
+            return final_url, {"Referer": REFERER}
+        except httpx.HTTPError as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)